### PR TITLE
Expand `Building/Site`

### LINKED ISSUE #300: Add Cambium GEA element
See https://www.nrel.gov/docs/fy22osti/81611.pdf. There is already an [eGRID Region](https://hpxml.nrel.gov/datadictionary/3.0.0/Building/Site/eGridRegion) element that this would live next to.

### PR DESCRIPTION
**Pull request by shorowit**
- Adds `GeoLocation` element, with `Latitude`, `Longitude`, `PlusCode`, `UBID`, and `extension` children
- Adds "Puerto Rico" as an additional `eGridRegion` enumeration choice
- Adds `eGridSubregion` element
- Adds `CambiumRegionGEA` element (closes #300)
- Adds `TimeZone` element, with `Name`, `Abbreviation`, `UTCOffset`, `DSTObserved`, and `extension` children

HPXMLBaseElements_Site

### CODE REVIEW

File: schemas/HPXMLBaseElements.xsd [hunk 1]
```diff
@@ -5237,7 +5237,45 @@
...
+									<xs:element minOccurs="0" name="Abbreviation" type="HPXMLString">
+										<xs:annotation>
+											<xs:documentation>For example, ET for Eastern Time.</xs:documentation>
+										</xs:annotation>
+									</xs:element>
+									<xs:element minOccurs="0" name="UTCOffset" type="HPXMLDouble">
+										<xs:annotation>
+											<xs:documentation>Positive or negative offset from Coordinated Universal Time (UTC). For example, -5 for Eastern Time.</xs:documentation>
+										</xs:annotation>
+									</xs:element>
```
Comment (nmerket) [inline schemas/HPXMLBaseElements.xsd:5275]: Should indicate standard time.

File: schemas/HPXMLBaseElements.xsd [hunk 2]
```diff
@@ -5237,7 +5237,45 @@
...
+							<xs:annotation>
+								<xs:documentation>For locations that observe Daylight Savings, use Standard time.</xs:documentation>
+							</xs:annotation>
+							<xs:complexType>
+								<xs:sequence>
+									<xs:element minOccurs="0" name="Name" type="HPXMLString">
+										<xs:annotation>
+											<xs:documentation>For example, Eastern Time.</xs:documentation>
+										</xs:annotation>
+									</xs:element>
```
Comment (nmerket) [inline schemas/HPXMLBaseElements.xsd:5265]: Provide a reference list here from like the Olsen database.

File: schemas/HPXMLBaseElements.xsd [hunk 3]
```diff
@@ -5237,7 +5237,45 @@
...
+							<xs:annotation>
+								<xs:documentation>Cambium Generation and Emission Assessment (GEA) region.</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+						<xs:element minOccurs="0" name="TimeZone">
+							<xs:annotation>
+								<xs:documentation>For locations that observe Daylight Savings, use Standard time.</xs:documentation>
+							</xs:annotation>
+							<xs:complexType>
+								<xs:sequence>
```
Comment (nmerket) [inline schemas/HPXMLBaseElements.xsd:5260]: Add `DSTObserved` boolean element.

**Review by nmerket — COMMENTED** [review]
From @charliepnnl, add a lat/lon element for geolocation. Add Google Place ID and UBID as well. Make a `GeoLocation` element for these to live in.

**Comment by nmerket** [comment]
Add `extension` to `TimeZone`

**Comment by shorowit** [comment]
I believe all review comments have been addressed. The PR description has been updated.

**Comment by nmerket** [comment]
🎥  [Discussion in Working Group Meeting 2022-05-13](https://www.youtube.com/watch?v=e7Ru1-PWJLI&t=880s)

**Comment by nmerket** [comment]
@nmerket to add documentation around google place id.

**Comment by nmerket** [comment]
🎥  [Discussion in HPXML Working Group Meeting 2022-05-27](https://www.youtube.com/watch?v=7mln2ynMEj4&t=1321s)

File: schemas/HPXMLBaseElements.xsd [hunk 4]
```diff
@@ -1,5 +1,6 @@
 <?xml version='1.0' encoding='UTF-8'?>
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns="http://hpxmlonline.com/2019/10" targetNamespace="http://hpxmlonline.com/2019/10" elementFormDefault="qualified" version="3.0">
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns="http://hpxmlonline.com/2019/10"
+	targetNamespace="http://hpxmlonline.com/2019/10" elementFormDefault="qualified" version="3.0">
```
[the remark this answers is not in the document]
Comment (nmerket) [reply schemas/HPXMLBaseElements.xsd:3]: No, I did not. I think that was Oxygen XML trying to be helpful.

[the remark this answers is not in the document]
Comment (nmerket) [reply schemas/HPXMLBaseElements.xsd:3]: I redid the formatting to remove all the extra stuff, but kept it for a few things I thought would be helpful, like to remove extra `</element>` tags when they're not necessary.

**Comment by nmerket** [comment]
🎥 [Final discussion in HPXML Working Group Meeting 2022-06-24](https://www.youtube.com/watch?v=XQyDFqjQoCM&t=90s)

**Pull request merged**